types/src/abi.rs

```diff
@@ -0,0 +1,675 @@
+use codec::{Decode, Encode};
```

Comment:
Is it a copy of t3rn-primitives? if so should be moved instead so there 1 implementation

Reply:
no copy, this has moved from t3rn-primitives